apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -2,89 +2,184 @@ defmodule AndiWeb.EditLiveView do
+              <%= Form.text_input(fp, :modifiedDate, class: "input") %>
+            </div>
+            <div class="metadata-form__spatial">
+              <%= Form.label(fp, :spatial, "Spatial Boundaries", class: "label") %>
+              <%= Form.text_input(fp, :spatial, class: "input") %>
+            </div>
+            <div class="metadata-form__temporal">
+              <%= Form.label(fp, :temporal, "Temporal Boundaries", class: "label") %>
+              <%= Form.text_input(fp, :temporal, class: "input") %>
+              <%= error_tag(fp, :temporal) %>
+            </div>
+            <div class="metadata-form__organization">
+              <%= Form.label(fp, :orgTitle, "Organization", class: "label label--required") %>
+              <%= Form.text_input(fp, :orgTitle, [class: "input input--text", disabled: true]) %>
+              <%= error_tag(fp, :orgTitle) %>
+            </div>
+            <div class="metadata-form__language">
+              <%= Form.label(fp, :language, "Language", class: "label") %>
+              <%= Form.select(fp, :language, [[key: "English", value: "english"], [key: "Spanish", value: "spanish"]], value: get_language(Form.input_value(fp, :language)), class: "select") %>
+            </div>
+            <div class="metadata-form__homepage">
+              <%= Form.label(fp, :homepage, "Data Homepage URL", class: "label") %>
+              <%= Form.text_input(fp, :homepage, class: "input") %>
+            </div>
+          <% end %>
+          <%= Form.inputs_for f, :technical, fn fp -> %>
+            <div class="metadata-form__format">
+              <%= Form.label(fp, :format, "Format", class: "label label--required") %>
+              <%= Form.text_input(fp, :sourceFormat, [class: "input", disabled: true]) %>
+              <%= error_tag(fp, :sourceFormat) %>
+            </div>
+            <div class="metadata-form__level-of-access">
+              <%= Form.label(fp, :private, "Level of Access", class: "label label--required") %>
+              <%= Form.select(fp, :private, [[key: "Private", value: "true"], [key: "Public", value: "false"]], class: "select") %>
+              <%= error_tag(fp, :private) %>
+            </div>
+          <% end %>
+        <div class="metadata-form__cancel-btn">
+          <%= Link.button("Cancel", to: "/", class: "btn btn--cancel") %>
+        </div>
+        <div class="metadata-form__save-btn">
+          <%= Form.submit("Save", id: "save-button", class: "btn btn--save", disabled: false) %>
+        </div>
+        </div>
+      <div>
+        <%= if @is_saved do %>
+          <div id="success-message" class="div__success-message">Saved Successfully</div>
+        <% end %>
+      </div>
+
     """
   end
 
   def mount(%{dataset: dataset}, socket) do
-    {:ok, assign(socket, dataset: dataset)}
+    new_business = dataset.business |> Map.from_struct()
+    new_technical = dataset.technical |> Map.from_struct()
+
+    change =
```

Comment:
I would put this pipe into `Andi.DatasetSchema`. Maybe into the changeset function or perhaps a different changeset function (having multiple special changeset functions seems to be idiomatic)